app/static/src/app/components/options/GraphSettings.vue

```diff
@@ -8,6 +8,14 @@
         <input type="checkbox" class="form-check-input" style="vertical-align:bottom;" v-model="logScaleYAxis">
       </div>
     </div>
+    <div id="lock-axes" class="row my-2">
+      <div class="col-5">
+        <label class="col-form-label">Lock axes</label>
```

Comment:
```suggestion
        <label class="col-form-label">Lock y axis range</label>
```

Reply:
as discussed, this actually locks both axes (for example when someone zooms in on the graph), if i dont lock both then the x axis autoscales when youre zoomed in which gives the graph a stretch that wasn't there before!